index.html

```diff
@@ -1,57 +1,134 @@
 <!DOCTYPE html>
 <html lang="en">
 
-	<!-- a head for all the heady/conceptual/meta stuff -->
+
 	<head>
 
-		<title><!-- your title goes here --></title>
+		<title>Cameron Priestley</title>
+
+
+		<script language="javascript" type="text/javascript" src="../p5/p5.min.js"></script>
+		<script language="javascript" type="text/javascript" src="../p5/addons/p5.dom.js"></script>
 
-		<!-- this imports p5  -->
-		<script language="javascript" type="text/javascript" src="p5/p5.min.js"></script>
-		<script language="javascript" type="text/javascript" src="p5/addons/p5.dom.js"></script>
 
-		<!-- here's your sketch! -->
 		<script>
-		// we're in javascript land now!
-		// (notice the different style of comments)
 
-		// here is where you declare your vars, like:
-		// var one = 1;
+		var currentLine = 0;
+
+		var totalLines = 327.3;
+
+		var output = "";
+
+
+
 
-	 /**
-	 * Setup - this is where you get things ready before running the loop
-	 */
 		function setup() {
-			// this tells p5 that we are not using the
-			// `<canvas>` element (more on that later)
-			noCanvas();
+			canvas
```

Comment:
This should use `noCanvas()` as in the original version. What you have here doesn't really do anything...